### ISSUE TITLE
W3C validation error using b-tabs/b-tab

### ISSUE THREAD

**Issue by jenemde** [bootstrap-vue/bootstrap-vue#5054]
### Describe the bug

If I use th b-tabs/b-tab component for creating a tabulator, I get error messages when validating the generated code:
- Attribute aria-setsize not allowed on element a at this point.
- Attribute aria-posinset not allowed on element a at this point.

These messages are repeated for every tab.
I get the same error messages validating this url:
https://bootstrap-vue.org/docs/components/tabs

### Steps to reproduce the bug

1. Go to https://bootstrap-vue.org/docs/components/tabs
2. Use bookmarklet "Check serialized DOM of current page" (from: https://validator.w3.org/nu/about.html)
3. Use wcag-bookmarklet "Check for WCAG 2.0 parsing compliance" (from: https://validator.w3.org/nu/about.html) on the result list
4. You now see the errors which mark the page as invalid in a wcag test

### Expected behavior

No errors in the tab module are shown.

### Versions

**Libraries:**

- BootstrapVue: 2.1.0
- Bootstrap: 4.3.1
- Vue: 2.6.10

**Environment:**

any device, any browser

**Comment by tmorehouse** [comment]
Strange, as the [WAI-ARIA `1.1`](https://www.w3.org/TR/wai-aria-1.1) spec shows `aria-posinset` and `aria-setsize` as being valid on `role="tab"`

https://www.w3.org/TR/wai-aria-1.1/#aria-posinset

**Comment by tmorehouse** [comment]
I've checked with Chrome Lighthouse, and I do not see any errors regarding `aria-posinset` or `aria-setsize`

What system are you using to test ARIA?

**Comment by tmorehouse** [comment]
It appears there is a bug in Nu Html Checker, as `aria-posinset` and `aria-setsize` are valid on elements that have `role="tab"`

**Comment by jenemde** [comment]
Thanks for your research.

Some more things I found out:
- bootstrap doesn't use the aria-attributes posinset and setsize (https://getbootstrap.com/docs/4.4/components/navs/#tabs)
- WAI-aria-best-practices doesn't us them
- WAI-aria also says: "Not required if all elements in the set are present in the DOM." (https://www.w3.org/TR/wai-aria-1.1/#aria-setsize and https://www.w3.org/TR/wai-aria-1.1/#aria-posinset)
- The error message says "Attribute aria-posinset not allowed on element a at this point". Don't know if it helps to have the attributes on the li-element (which then also needs the tab-role) instead of the anchor.

But yes, I'm also astonished that the Nu Html Checker marks this as an error. Maybe it's possible to add a property to set or unset the attributes for the user of the component.

**Comment by tmorehouse** [comment]
- Bootstrap v4 doesn't always show all ARIA attributes they should be using (there are a few open/closed issues about this)
- Tabs can be lazy rendered (i.e. not in the DOM until shown, then removed when hidden)
